### DISCUSSION TITLE
Nuxt Content pages do not show up on Google and have no embed on social media

### DISCUSSION THREAD

**Discussion by JannisX11** [nuxt/nuxt#10772] [Questions (Nuxt 2)] [answered]
I have a website with a wiki that is built using Nuxt Content and .md files. However none of the wiki pages show up on Google, which is one of the main points of the wiki. Also, embeds on social media and Discord do not work for any links from the Wiki. They do work for the rest of the website.

* Wiki page for demonstration: https://www.blockbench.net/wiki/blockbench/themes
* Nuxt v2.15.7, Nuxt Content v1.14.0
* Hosted on Netlify
* Target is static and SSR is false
* Source code: https://github.com/JannisX11/blockbench.net/blob/main/pages/wiki/_.vue
* Config: https://github.com/JannisX11/blockbench.net/blob/main/nuxt.config.js

What I've tried:
* Enabling SSR in the nuxt config. However doing that breaks the downloads page (https://github.com/JannisX11/blockbench.net/blob/main/pages/downloads.vue), which needs to fetch the correct version client-side on page load. Also, it does not seem to help with any of the issues.

**Comment by TheAlexLichter** [comment, marked as answer]
With `SSR: false`, no meta-tags will be part of the HTML. Thus, there is no share preview or embed on social media (except possibly defaults). There is no way around it except using some way of SSR (for example SSG).

PS: You do not need to set `target: static` in Nuxt 3.

**Reply by JannisX11** [reply]
Thanks. I tried static site generation initially, but it didn't work. Then I investigated, and found out that Nuxt never generated any content pages. This was because it uses a Crawler to find the pages to generate, and there were no links to content pages except dynamically generated ones, so it didn't find the pages. The solution I used for now was to add those pages to the generate.routes property in nuxt config. In the future this could be solved by manually writing the navigation sidebar, instead of generating it dynamically. That way it would include all routes already.

**Comment by danielroe** [comment]
Just FYI, this is the repository for Nuxt 3, and you are using Nuxt 2.

**Reply by JannisX11** [reply]
I thought [nuxt/framework](https://github.com/nuxt/framework) was the repository for Nuxt 3?

**Reply by danielroe** [reply]
Yes, it is.

**Reply by danielroe** [reply]
(I transferred this issue back to `nuxt/nuxt.js`.)

**Reply by JannisX11** [reply]
Ah, cool cool!
